docker-compose.yml

```diff
@@ -15,12 +15,13 @@ services:
       PYTHONUNBUFFERED: 1
       PYTHONDONTWRITEBYTECODE: 1
 
-  nylas:
+  app:
     build:
       context: .
       dockerfile: Dockerfile
     volumes:
-      - .:/src
+      - .:/opt/app
+    tmpfs: /opt/app/inbox_sync.egg-info
```

Comment:
I had to do this so `pip install -e .` keeps working after mounting the source code volume now that we use non-root user and Docker on Linux mounts volumes as root: https://github.com/moby/moby/issues/2259

Another alternative would be to chown to 5000:5000 the code on the host but I found that ugly.